<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trainer</class>
 <widget class="QMainWindow" name="Trainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trainer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="trainer_tb_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>751</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="locale">
     <locale language="Spanish" country="Spain"/>
    </property>
    <property name="documentTitle">
     <string>Reading</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trainer_btn_record">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trainer_btn_cancel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="trainer_lbl_training_session">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Training session</string>
    </property>
   </widget>
   <widget class="QLabel" name="training_lbl_general_indications">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>481</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>After recording a training, you can reset it or cancel it</string>
    </property>
   </widget>
   <widget class="QLabel" name="training_lbl_save_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>511</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Your training is ready to be saved, to save it just click in 'Save'</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trainer_btn_save">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="training_lbl_recording">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Recording:</string>
    </property>
   </widget>
   <widget class="QLabel" name="training_lbl_recording_seconds">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
